javaslang/src/test/java/javaslang/collection/VectorTest.java

```diff
@@ -208,46 +205,4 @@ public void shouldStringifyNil() {
     public void shouldStringifyNonNil() {
         assertThat(of(null, 1, 2, 3).toString()).isEqualTo("Vector(null, 1, 2, 3)");
     }
-
```

Comment:
these were testing `List`, not applicable to `Vector`, since its `Empty` is not a separate class